lib/src/game/game_core.dart

```diff
@@ -19,9 +19,9 @@ class Game {
   final _gameStateEvent = StreamController<GameState>();
   final _watch = Stopwatch();
 
-  GameState _state;
-  int _bombsLeft;
-  int _revealsLeft;
+  late GameState _state;
```

Comment:
This is initialized in the constructor initializer list, does it need to be late?  If not, should there be a hint/lint for "unnecessary late field"?

Reply:
Ooo...let me look

Reply:
https://github.com/dart-lang/sample-pop_pop_win/commit/7eb9ef343758f131a084915901dc1487b9ab1f52

Turns out I can do it with _bombsLeft and _revealsLeft, too!